docker/Dockerfile.customopbuilder.clean

```diff
@@ -56,4 +56,12 @@ RUN rm -f /usr/bin/python && \
     pip --version
 
 COPY --from=cuda /usr/local/cuda /usr/local/cuda
-ENV LD_LIBRARY_PATH=/usr/local/cuda/lib64:/usr/local/cuda/lib64/stubs/:${LD_LIBRARY_PATH}
\ No newline at end of file
+ENV LD_LIBRARY_PATH=/usr/local/cuda/lib64:/usr/local/cuda/lib64/stubs/:${LD_LIBRARY_PATH}
+
+WORKDIR /opt/dali
+COPY qa/setup_packages.py qa/setup_packages.py
+RUN export USE_CUDA_VERSION=$(cat /usr/local/cuda/version.txt | head -1 | sed 's/.*Version \([0-9]\+\)\.\([0-9]\+\).*/\1\2/') && \
+    export last_config_index=$(python qa/setup_packages.py -n -u tensorflow-gpu --cuda ${USE_CUDA_VERSION}) && \
```

Comment:
Could you put together one line of comment, what this layer is supposed to do? It's quite enigmatic with all the `sed` calls and for loops ;)

Reply:
Done